File: Iniciante/1098 - Sequencia IJ 4/Python/1098.py
def main():
    for k in range(11):
        i = k * 0.2
        for j in range(1, 4):
            valueJ = j + i
            if i == 0 or i == 1 or i == 2:
                print(f"I={i:.0f} J={valueJ:.0f}")
            else:
                print(f"I={i:.1f} J={valueJ:.1f}")

if __name__ == "__main__":
    main()

File: Iniciante/1098 - Sequencia IJ 4/JavaScript/1098.js
function main() {
    for (let k = 0; k <= 10; k++) {
        let i = k * 0.2;
        for (let j = 1; j <= 3; j++) {
            let valueJ = j + i;
            if (i === 0 || i === 1 || i === 2) {
                console.log(`I=${i.toFixed(0)} J=${valueJ.toFixed(0)}`);
            } else {
                console.log(`I=${i.toFixed(1)} J=${valueJ.toFixed(1)}`);
            }
        }
    }
}

main();